members/models/person.py

```diff
@@ -220,11 +228,15 @@ def update_dawa_data(self):
                     dawa_id=access_address["kommune"]["kode"]
                 )
                 self.dawa_id = address["id"]
-                self.save()
+                if save:
+                    self.save()
+                else:
+                    return self
```

Comment:
Vil vi ikke gerne altid returnere `self` her, uanset om der gemmes eller ej?
```suggestion
                if save:
                    self.save()
                return self
```